.gitignore

```diff
@@ -1,3 +1,53 @@
 target/
 **/*.rs.bk
 Cargo.lock
+
+### macOS ###
```

Comment:
please revert these additions, they can be added to a users global `gitignore`

Reply:
Done with 02b5f8d